packages/kubernetes-common/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/kubernetes-common",
```

Comment:
Based on https://backstage.io/docs/architecture-decisions/adrs-adr011, this should actually be in `plugins/kubernetes-common` and the package should have the name `@backstage/plugin-kubernetes-common`?

Reply:
That makes a _lot_ of sense. It was weird to have a "kubernetes-common" given it's just about the plugin, not kube in general.